freqtrade/freqtradebot.py

```diff
@@ -904,6 +904,7 @@ def handle_timedout_limit_buy(self, trade: Trade, order: Dict) -> bool:
             logger.info('Buy order %s for %s.', reason, trade)
 
         if corder.get('remaining', order['remaining']) == order['amount']:
+            logger.info('Buy order removed from database %s', trade)
```

Comment:
should this message be better placed after session.delete()?

Reply:
well no - then you can't log "trade" anymore reliably (depending on the SQLalchemy backend i think) as it's technically gone (deleted)... 
Also, that would assume that something would be changed on the trade while deleting it - which isn't the case. It's just "informative" - so the logs contain "this has been removed!".

Reply:
I mean, the sentence "... removed from database" is printed before removing, should either be after actual removal or print "removing..."